pysteps/nowcasts/interface.py

```diff
@@ -33,15 +33,19 @@
 
 from pysteps.extrapolation.interface import eulerian_persistence
 from pysteps.nowcasts import anvil, sprog, steps, sseps, extrapolation
+from pysteps.nowcasts import lagrangian_probability
 
 _nowcast_methods = dict()
 _nowcast_methods["anvil"] = anvil.forecast
 _nowcast_methods["eulerian"] = eulerian_persistence
 _nowcast_methods["extrapolation"] = extrapolation.forecast
 _nowcast_methods["lagrangian"] = extrapolation.forecast
+_nowcast_methods["probability"] = lagrangian_probability.forecast
+_nowcast_methods["lagrangian_probability"] = lagrangian_probability.forecast
 _nowcast_methods["sprog"] = sprog.forecast
 _nowcast_methods["sseps"] = sseps.forecast
 _nowcast_methods["steps"] = steps.forecast
+_nowcast_methods["steps"] = steps.forecast
```

Comment:
If the review window is correct, the two lines above are the same.

Reply:
well spotted! done in 4cd86b3